Model/Page.php

```diff
@@ -134,6 +134,11 @@
     protected $target;
 
     /**
+     * @var int
+     */
+    protected $targetRedirectType = 302;
```

Comment:
I agree with this. But I think it's a little weird as this will be the only field which will be private, not that it really matters. I guess changing all to private will introduce more BC-breaks and should be targeted for the next major version.